### PR TITLE
First alpha version of the library

### PR DESCRIPTION
**Pull request by rafaelje**
## Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: [Conventional Commit](https://conventionalcommits.org/)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [x] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the new behavior?

This is the first alpha version of the library with:
- initial documentation
- initial instrumented test
- Java Example
- CI
- initsession()
- killsession()
- getMyProfiles()
- getActiveProfile()
- changeActiveProfile()
- lostPassword()
- recoveryPassword()
- getMyEntities()
- getActiveEntities()
- changeActiveEntities()
- getFullSession()
- getAnItem()
- getAllItems()
- getSubItems()
- addItems()
- updateItems()
- deleteItems()

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by ajsb85 — APPROVED** [review]
Reviewed

File: glpi/src/main/java/org/flyve/glpi/query/GetSubItemQuery.java [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+*  --------------------------------------------------------------------
+*  @author    Rafael Hernandez - <rhernandez@teclib.com>
+*  @copyright (C) 2017 Teclib' and contributors.
+*  @license   GPLv3 https://www.gnu.org/licenses/gpl-3.0.html
+*  @link      https://github.com/glpi-project/java-library-glpi
+*  @link      http://www.glpi-project.org/
+*  --------------------------------------------------------------------
+*/
+
+package org.flyve.glpi.query;
```
Comment (ajsb85) [inline glpi/src/main/java/org/flyve/glpi/query/GetSubItemQuery.java]: Why org.flyve.glpi?

The package id is: org.glpi.api

**Comment by rafaelje** [comment]
@ajsb85 When you can Please help me to review this PR

**Review by ajsb85 — CHANGES_REQUESTED** [review]

**Comment by ajsb85** [comment]
This branch cannot be rebased due to conflicts

**Review by ajsb85 — APPROVED** [review]
Reviewed

**Review by ajsb85 — APPROVED** [review]
Reviewed

**Comment by ajsb85** [comment]
Hi, @rafaelje 

Could you rebase your branch to resolve the conflicts in the PR?

Thanks.

**Comment by ajsb85** [comment]
Hi, @rafaelje This branch cannot be rebased due to conflicts

I can see the changes to update in your branch here:

https://github.com/rafaelje/java-library-glpi/compare/develop...glpi-project:develop?expand=1

**Comment by rafaelje** [comment]
Hi @ajsb85 you are trying to compare develop branch with develop but is feature/api with develop

**Pull request closed without merging**